File: src/net/merayen/elastic/uinodes/list/output_1/Info.java
package net.merayen.elastic.uinodes.list.output_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override
	public String getName() {
		return "Output";
	}

	@Override
	public String getDescription() {
		return "Outputs audio to speakers";
	}

	@Override
	public String[] getCategories() {
		return new String[]{"Routing"};
	}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/frequency_1/Info.kt
package net.merayen.elastic.uinodes.list.frequency_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "Frequency"
	override fun getDescription() = "Analyze frequencies and output midi"
	override fun getCategories() = arrayOf("Audio")
}